extensions/dockerhub/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "dockerhub",
+  "title": "dockerhub",
+  "description": "Search images on DockerHub.",
+  "icon": "docker-icon.png",
+  "author": "k8scat",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "search-image",
+      "title": "Search Images",
+      "subtitle": "Images on Docker Hub",
+      "description": "Search images on Docker Hub",
```

Comment:
I'd suggest to use "Search Docker Images" as title and leave the subtitle empty. Also see our store guidelines: https://developers.raycast.com/basics/prepare-an-extension-for-store